### PR TITLE
Change basic output methods names and fix comment typos

### PR DESCRIPTION
**Pull request by kmmelcher**
I think this minor change improves code readability

### CODE REVIEW

**Comment by cuducos** [comment]
Honestly, I disagree.

Surely it is open for debate, but I do like the childish creativity of the way these methods are named and I argue that they are more readable (not less readable), for example:

* Ignoring the `self.`, if you read this line out loud it sounds just like plain English: `self.oops("To use putgist you have to set your GETGIST_TOKEN")` — it literally reads _oops, to use `putgist` you have to set your GetGist token_; on the other hand, _error message to use `putgist` you have to set your GetGist token_ is not a grammatical correctly sentence in plain English.

The same is valid for other examples as well, such as _hey, the URL of the Gist is blah blah_ or _yeah, user authenticaded_. Theses sentences are barely the same in plain English and in the source code and we loose that because _highlight message the URL is_ doesn't read that fluent in English.

Surely that is a damn opinionated argument. But I would be happy to hear more about why you think these name changes actually improve code readability.

**Comment by kmmelcher** [comment]
Okay, sorry

**Comment by cuducos** [comment]
No need to be sorry, and maybe no need to close the PR. It's an open discussion : ) Feel free to share your thoughts if you like (and, if that's the case, re-open the PR).

**Comment by cuducos** [comment]
And, for sure, the typos are still a valid contribution. I am a master of typos :facepalm:
